openslides_backend/action/actions/user/json_upload.py

```diff
@@ -0,0 +1,167 @@
+                    "properties": {
+                        **model.get_properties(
+                            "title",
+                            "first_name",
+                            "last_name",
+                            "is_active",
+                            "is_physical_person",
+                            "default_password",
+                            "email",
+                            "username",
+                            "gender",
+                            "pronoun",
+                        ),
+                    },
+                    "required": [],
+                    "additionalProperties": False,
+                },
+                "minItems": 1,
+                "uniqueItems": False,
+            },
+        }
+    )
+    headers = [
+        {"property": "title", "type": "string"},
+        {"property": "first_name", "type": "string"},
+        {"property": "last_name", "type": "string"},
+        {"property": "is_active", "type": "boolean"},
+        {"property": "is_physical_person", "type": "boolean"},
+        {"property": "default_password", "type": "string"},
+        {"property": "email", "type": "string"},
+        {"property": "username", "type": "string"},
+        {"property": "gender", "type": "string"},
+        {"property": "pronoun", "type": "string"},
+    ]
+    permission = OrganizationManagementLevel.CAN_MANAGE_USERS
+    skip_archived_meeting_check = True
+
+    def update_instance(self, instance: Dict[str, Any]) -> Dict[str, Any]:
+        data = instance.pop("data")
+
+        # validate and check for duplicates
+        self.init_duplicate_set(
+            [
+                {
+                    field: entry.get(field, "")
+                    for field in ("username", "first_name", "last_name", "email")
+                }
+                for entry in data
+            ]
+        )
+        rows = [
+            self.generate_entry(entry, payload_index)
+            for payload_index, entry in enumerate(data)
+        ]
+
+        self.init_rows(rows)
+        self.store_rows_in_the_action_worker("account")
+        return {}
+
+    def generate_entry(
+        self, entry: Dict[str, Any], payload_index: int
+    ) -> Dict[str, Any]:
+        status, error = None, []
+        try:
+            UserCreate.schema_validator(entry)
+            if entry.get("username"):
+                if self.check_username_for_duplicate(entry["username"], payload_index):
+                    status = ImportStatus.DONE
+                    if searchdata := self.get_search_data(payload_index):
+                        entry["id"] = searchdata["id"]
+                    else:
+                        status = ImportStatus.ERROR
+                        error.append(f"Duplicate in csv list index: {payload_index}")
+                else:
+                    status = ImportStatus.NEW
+                entry["username"] = {"value": entry["username"], "info": "done"}
```

Comment:
```suggestion
                entry["username"] = {"value": entry["username"], "info": ImportStatus.DONE}
```
Also use the enum everywhere else

Reply:
Done.